src/lib/axios.js

```diff
@@ -0,0 +1,33 @@
+function init({ baseURL = BASE_URL, axiosOptions = { headers: {} } } = {}) {
```

Comment:
If you are adopting the arrow function syntax, which is recommended for its concise and clear structure, please ensure to use this pattern consistently across all function declarations in the codebase.